### DISCUSSION TITLE
올라잇 : 청각장애인의 보이는 목소리, 대화

### DISCUSSION THREAD

**Discussion by yoondj98** [DeveloperAcademy-POSTECH/.github#57] [Show and tell]
안녕하세요! Xerath입니다! 👋🏻 🤖 👋🏻
이번 MC3 프로젝트에서 Team.10구만 팀원으로 참여하여 개발한 앱 **올라잇**을 소개합니다!

저는 카페에서 에어팟을 끼고 Noise Cancelling을 한 뒤 과외를 하다보면 자연스레 소리가 올라가 주변에서 시선을 받아본 경험이 있습니다.
그 이유를 고민해보니 에어팟으로 인해 제 목소리도 필터링이 되어 제 목소리가 큰 것을 인지하지 못했던 부분이었습니다.

이러한 경험을 청각장애인 분들 입장에서도 아주 유사한 경험으로 갖고 계시다는 것을 알게 되어,
나의 목소리를 눈으로 확인 가능하도록 하여 자연스레 조절을 할 수 있는 앱을 만들게 되었습니다! 😄😄
많은 관심 부탁드립니다! :-D

👉🏻 👉🏻 [**iOS 앱 설치**](https://apps.apple.com/kr/app/%EC%98%AC%EB%9D%BC%EC%9E%87-%EB%B3%B4%EC%9D%B4%EB%8A%94-%EB%AA%A9%EC%86%8C%EB%A6%AC-%EB%8C%80%ED%99%94/id6593663549)

<img width="250" src="https://github.com/user-attachments/assets/b4107161-6942-43cb-baff-6ca1df84ecfe" />
<img width="250" src="https://github.com/user-attachments/assets/6e8c2f17-2e3c-4a3f-8629-252e918cb78c" />
<img width="250" src="https://github.com/user-attachments/assets/d9068f62-b220-4af6-94dc-2ee3cb99ec2d" />

<img width="250" src="https://github.com/user-attachments/assets/61c05ccc-090b-4f5e-bb18-2537c02e4a30" />
<img width="250" src="https://github.com/user-attachments/assets/1f631696-b149-4200-a3b1-da3696bcff09" />

**Comment by SANGDOLEE** [comment]
최고의 앱 !

**Comment by yeniful** [comment]
짱이에요!!!
